modules/statics/src/networks.ts

```diff
@@ -1388,8 +1388,8 @@ class MonadTestnet extends Testnet implements EthereumNetwork {
 class Monad extends Mainnet implements EthereumNetwork {
   name = 'Monad';
   family = CoinFamily.MON;
-  explorerUrl = 'https://testnet.monadexplorer.com/tx/'; //TODO: WIN-5225: change it with mainnet explorer, when its available
-  chainId = 10143; //TODO: WIN-5225: change it with mainnet explorer, when its available
+  explorerUrl = 'https://mainnet-beta.monvision.io/';
```

Comment:
This link is asking for username and password. Hope that should be OK?
This is usually a transaction explorer
```suggestion
  explorerUrl = 'https://mainnet-beta.monvision.io/tx/';
```